example/MyConfigDefinition.php

```diff
@@ -1,7 +1,15 @@
-<?php
+<?php declare(strict_types = 1);
+
+namespace MyComponent;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\NodeDefinition;
 
 class MyConfigDefinition extends \Keboola\Component\Config\BaseConfigDefinition
 {
+    /**
+     * @return ArrayNodeDefinition|NodeDefinition
+     */
```

Comment:
do we need ArrayNodeDefinition here? parameters is also always an object (or should be normalized to), so `NodeDefinition` should be enough

Reply:
Now that I think about it... is should be always `ArrayNodeDefinition`.  Because the parameters are always array. If it were `NodeDefinition`, it would fail because `children()` is does not exist there :(

Reply:
ok, i always mix the classes, but the point is that only one of them is sufficient